src/bandersnatch_storage_plugins/filesystem.py

```diff
@@ -138,6 +134,13 @@ def copy_file(self, source: PATH_TYPES, dest: PATH_TYPES) -> None:
         shutil.copy(source, dest)
         return
 
+    def move_file(self, source: PATH_TYPES, dest: PATH_TYPES) -> None:
+        """Move a file from **source** to **dest**"""
+        if not self.exists(source):
+            raise FileNotFoundError(source)
+        shutil.move(str(source), dest)
```

Comment:
No biggy, but I think >= 3.8 move can handle PATH_TYPES ... (If wrong just ignore)

Reply:
I tried on Python 3.9.2 and my tox run still shout at me about it so I added it...

Reply:
Looking at the Python docs, this is expected behaviour:

> Changed in version 3.9: Accepts a path-like object for both src and dst.

https://docs.python.org/3/library/shutil.html#shutil.move

Our mypy configuration is configured to have mypy believe we are writing a Python 3.8 project.

https://github.com/pypa/bandersnatch/blob/164616ecb611714e9a0c410c8feb2ab86b0d5edd/mypy.ini#L2

Reply:
Cool - thought it was 3.8 - my bad. This is all doing the correct things then! Thanks for checking all.